manifests/init.pp

```diff
@@ -361,6 +361,8 @@
   Boolean $templates = $foreman_proxy::params::templates,
   Foreman_proxy::ListenOn $templates_listen_on = $foreman_proxy::params::templates_listen_on,
   Stdlib::HTTPUrl $template_url = $foreman_proxy::params::template_url,
+  Boolean $registration = $foreman_proxy::params::registration,
+  Foreman_proxy::ListenOn $registration_listen_on = $foreman_proxy::params::registration_listen_on,
```

Comment:
If you refer to variables in params, the need to be defined in params.pp as well.